crates/ra_hir/src/ty/tests.rs

```diff
@@ -116,6 +116,27 @@ fn test(a: &u32, b: &mut u32, c: *const u32, d: *mut u32) {
     );
 }
 
+#[test]
+fn infer_literals() {
+    check_inference(
+        r#"
+fn test() {
+    5i32;
+    "hello";
+    b"bytes";
+    'c';
+    b'b';
+    3.14;
+    5000;
+    (0u32, -5isize);
+    false;
+    [true, true, false]
```

Comment:
maybe add a semicolon here so it's not unified with the return type `()` ;)

Reply:
Woops!